src/hokusai/hokusai.yml

```diff
@@ -56,9 +56,19 @@ commands:
       - run:
           name: Configure Hokusai
           command: |
-            mkdir -p ~/.hokusai
-            curl -o ~/.hokusai/config.yml << parameters.configUri >>
-            hokusai configure
+            # sed removes color code from hokusai output
+            version=$(hokusai version | sed "s,\x1B\[[0-9;]*[a-zA-Z],,g")
+            echo "hokusai version: $version"
+            major=${version:0:1}
+
+            if [ "$major" -lt "2" ]
```

Comment:
while this should work for a canonical Hokusai v2 release, it doesn't work with a Hokusai beta release. Hokusai beta releases [don't bump major version](https://app.circleci.com/pipelines/github/artsy/hokusai-sandbox/614/workflows/cc6a30d1-2b77-4a19-9182-361c44ccfed0/jobs/855?invite=true#step-104-0_36).

Reply:
so projects that are on Hokusai beta (Gravity/Force/Volt) will break.

Reply:
[solved](https://github.com/artsy/orbs/pull/158/commits/3b3284c6647dfcaf2b309341b87a5e30401bc9ec).